third_party/move/documentation/coding_guidelines.md

```diff
@@ -23,6 +23,29 @@ $ cargo xclippy --all-targets
 
 In general, we follow the recommendations from [rust-lang-nursery](https://rust-lang.github.io/api-guidelines/) and [The Rust Programming Language](https://doc.rust-lang.org/book/).  The remainder of this guide provides detailed guidelines on specific topics in order to achieve uniformity of the codebase.
 
+## Code review
+
+We require every PR to have approval by at least two reviewers. This is enforced by the CI presubmit.
+
+### Guidelines for Authors of PRs
+
+- Have a meaningful description of the PR
+    - the purpose of a PR
+    - major design decisions
+    - testing strategy
+    - any additional, non-primary changes related to the PR. It is OK to do minor refactorings of the code as you visit it.
+- Respond to each comment of the reviewers. If you think you resolved a request for a change, just use 'done' as a response. If you diasgree with the reviewer, explain why
+- Once you are done with addressing comments, indicate this to the reviewers by adding a top-level comment to the PR. You can use 'PTLA' ('please take a look again') as an acronym for this purpose.
+- If possible, avoid force push so the reviewer can see how you changed code in comparison. There are exceptions to this, for example if you need to rebase
```

Comment:
Should we add any specifics about individual commits? AWS had strict rules about how each significant change should be a separate commit, and also certain rules and guidelines around rebasing.

Reply:
We don't have such strict policies intentionally (we are a startup not a big corp and pace is important -- though many of us come from Meta/Google with similar policies), but there is a point to not mixing totally independent things. I added: "Avoid merging independent things into one PR. (Except minor refactoring and bug fixes as mentioned above.)"